src/Bridge/Symfony/Bundle/DependencyInjection/ApiPlatformExtension.php

```diff
@@ -86,6 +86,11 @@ public function load(array $configs, ContainerBuilder $container)
         if ($config['enable_fos_user']) {
             $loader->load('fos_user.xml');
         }
+
+        // NelmioApiDoc support
+        if ($config['nelmio_api_doc']) {
```

Comment:
FWIW, we have the `enable_fos_user` option. Do we want the user to be able to control this explicitly (e.g. continue using `NelmioApiDocBundle` but without the integration from API Platform)?